layouts/partials/breadcrumb.html

```diff
@@ -1,4 +1,20 @@
-<nav aria-label="breadcrumb" class="d-flex align-items-center bg-light bg-gradient border-bottom px-3 py-2 shadow-sm">
+<nav aria-label="breadcrumb" class="d-flex align-items-center bg-light bg-gradient border-bottom px-3 py-1 shadow-sm">
+
+  {{ $selected_dialect := .dialect }}
+  <div class="dropdown pe-3">
+    <button class="btn btn-outline-primary dropdown-toggle btn-sm" type="button" data-bs-toggle="dropdown" aria-expanded="false">
+      Other dialects
+    </button>
+    <ul class="dropdown-menu">
+      {{ range .siteSections }}
+        {{ if ne $selected_dialect .Title }}
+          <li><a class="dropdown-item disabled" aria-disabled="true" href="{{ ref . .RelPermalink }}">{{ .Title }}</a></li>
+        {{ end }}
+      {{ end }}
+      <li><a class="dropdown-item"> More coming soon! </a></li>
```

Comment:
Also, you added spaces before/after the square brackets, which is not the case elsewhere:

```suggestion
      <li><a class="dropdown-item">More coming soon!</a></li>
```

Reply:
@jviotti I have made the changes.